doc/reference/storage/settings/settings.rst

```diff
@@ -69,9 +69,10 @@ backend.
 Zephyr Storage Backends
 ***********************
 
-Zephyr has two existing backend storages which can be a Flash Circular Buffer
-(:option:`CONFIG_SETTINGS_FCB`) or a file in the filesystem
-(:option:`CONFIG_SETTINGS_FS`).
+Zephyr has three existing backend storages which can be a Flash
```

Comment:
Let's change the beginning of the sentence to:
```
Zephyr has three storage backends: a Flash
```

Reply:
@dbkinder Thanks, I have applied your suggestion.